docs/source/getting_started/index.rst

```diff
@@ -160,14 +134,13 @@ let's get familiar with ``iroha-cli``
 
   <script src="https://asciinema.org/a/6dFA3CWHQOgaYbKfQXtzApDob.js" id="asciicast-6dFA3CWHQOgaYbKfQXtzApDob" async></script>
```

Comment:
Please update ascii cinema "video".

Reply:
I can provide source files and we can modify them :)